tools/intensity_check/xml_intensity_check.xml

```diff
@@ -0,0 +1,287 @@
++----------------------------+------------+
+| 3 : Variable metadata file |   tabular  |
++----------------------------+------------+
+
+----------
+
+Parameters
+----------
+
+**Computation method**  
+    - **Without distinction between samples:** calculates chosen statistic(s) for each variable.  
+    - **For each class of samples:** separates samples between each class (class column to be specified). Chosen statistic(s) and/or mean fold change are calculated for each of them.  
+    - **Between one class versus all the remaining samples:** If you want to focus only on one class versus all the remaining samples without class distinction.  
+
+    In the case of two classes: "each class" and "one class" give the same results for statistical measures. We recommend to choose "one class" for mean fold change calculation in order to select the class you want to put as numerator or denominator (see below).
+
+**Statistics**  
+    Select the statistical measures you want to add in the variable metadata table. If the method is "each class" or "one class", you can choose to leave this section blank if you only want to calculate the mean fold change (see below).
+
+**Class column** (*only if "each class" or "one class"*)  
+    Select the class column in sample metadata table.
+
+**Selected class** (*only if "one class"*)  
+    If the method is "one class", specify it. Remaining samples will be named "Other".
+
+**Calculate the mean fold change** (*only if "each class" or "one class"*)  
+    Choose if you want to calculate the mean fold change. If the method is "each class": mean fold change will be calculated for all combinations of classes. If the method is "one class": it will be calculated between the selected class (see above) and the remaining samples.
+
+**Where should the class be placed for the mean fold change calculation?** (*only if "one class"*)  
+    If the method is "one class", choose "top" or "bottom" to put the selected class as numerator or denominator (respectively) for the mean fold change calculation.
+
+**Logarithm** (*only if "each class" or "one class"*)  
+    Choose if you want to transform the mean fold change with a log2 or log10.
+
+------------
+Output files
+------------
+
+**Variable metadata file**  
+    Contains the previous columns in variable metadata file and the new ones.  
+    In the column names for fold, the first class specified is the one used as numerator in the ratio.
+
+**Graphs file**  
+    Contains barplots with the proportion of NA considering classes and boxplots with the fold values.
+
+---------------------------------------------------
+
+----------------------
+Additional information
+----------------------
+
+.. class:: warningmark
+
+For more information about input files, refer to the corresponding "W4M HowTo" page:  
+`W4M table format for Galaxy <https://workflow4metabolomics.org/sites/default/files/fichiers/documents/w4m_TableFormatForGalaxy_150908.pdf>`_
```

Comment:
The linting seems to indicate a problem with the 3-tables format weblink, wich is quite old in fact. Maybe with the following link it will be better?
```suggestion
`W4M table format for Galaxy <https://nextcloud.inrae.fr/s/qLkNZRf84QQ5YLY>`_
```